srcpkgs/glab/template

```diff
@@ -1,18 +1,24 @@
 # Template file for 'glab'
 pkgname=glab
-version=1.22.0
+version=1.24.1
 revision=1
 build_style=go
+build_helper=qemu
 go_ldflags="-X main.version=$version"
-go_import_path=github.com/profclems/glab
+go_import_path=gitlab.com/gitlab-org/cli
 go_package="${go_import_path}/cmd/glab"
 short_desc="Command line tool bringing GitLab's features to your command line"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="MIT"
 homepage="https://gitlab.com/gitlab-org/cli"
 distfiles="https://gitlab.com/gitlab-org/cli/-/archive/v$version/cli-v$version.tar.gz"
-checksum=4d9bceb6818c8bf9f681119dae3a65f1c895fa21e9da6b38e8f88d245f524e10
+checksum=dc942f7806aa417714483bd5323bfcde9eceadd7ed33154f7a77038b416bdd95
 
 post_install() {
+	local glab=${DESTDIR}/usr/bin/glab
+	for shell in bash fish zsh; do
+		vtargetrun ${glab} completion -s ${shell} > glab.${shell}
+		vcompletion glab.${shell} ${shell}
+	done
```

Comment:
there is no point in setting a variable when it's only going to be used once in the code
```suggestion
	for shell in bash fish zsh; do
		vtargetrun ${DESTDIR}/usr/bin/glab completion -s ${shell} > glab.${shell}
		vcompletion glab.${shell} ${shell}
	done
```